pkg/http/api/server.go

```diff
@@ -22,6 +24,29 @@ func NewServer(config etc.API, handler http.Handler) *Server {
 			WriteTimeout: config.WriteTimeout,
```

Comment:
Could we also add `IdleTimeout` to limit keep-alive? Maybe `120` as a value suggested by Cloudflare https://blog.cloudflare.com/exposing-go-on-the-internet/

Reply:
Good catch. I have it in the config struct, but did not use it here. Thanks!